GenerateToolingFeed/V4Format/V4FormatFeedEntryUpdater.cs

```diff
@@ -6,17 +6,30 @@ namespace GenerateToolingFeed.V4Format
 {
     internal class V4FormatFeedEntryUpdater : IFeedEntryUpdater
     {
-        private static readonly IDictionary<string, string> _dotnetToTemplatesPrefix = new Dictionary<string, string>()
+        private static readonly IDictionary<string, string[]> _dotnetToItemTemplatesPrefix = new Dictionary<string, string[]>()
```

Comment:
Don't need string array.  You can only have one item template per dotnet runtime / worker version
Use `IDictionary<string, string>` instead